.gitignore

```diff
@@ -50,9 +50,8 @@ site
 
 .turbo
 
-# Cypress
-**/cypress/downloads
-**/cypress/screenshots
+# e2e test results
+e2e-tests/junit-results.xml
```

Comment:
you may remove from root `.gitignore`
```
# Cypress
**/cypress/downloads
**/cypress/screenshots
```

but better we move  `junit-results.xml` to `e2e-tests/.gitignore`

Reply:
Yes, that makes sense. Thank you